testplan/cli/commands/writers.py

```diff
@@ -84,10 +88,16 @@ def __init__(self, port: int):
         self.port = port
 
     def __call__(self, result: TestReport) -> TestReport:
-
-        exporter = WebServerExporter(ui_port=self.port)
-        exporter.export(result)
-        exporter.wait_for_kb_interrupt()
+        with tempfile.TemporaryDirectory() as tmpdir:
+            json_path = os.path.join(tmpdir, "report.json")
+            exporter = JSONExporter(
+                json_path=json_path, split_json_report=False
+            )
+            exporter.export(result)
```

Comment:
This part is repeating WebServerExporter logic, I'm fine if we leave a comment saying this part is to be optimized.